lib/core/src/persist/migrations.rs

```diff
@@ -183,5 +183,26 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
 
         DROP TABLE old_chain_swaps;
         ",
+        "CREATE TABLE IF NOT EXISTS sync_state(
+            data_id TEXT NOT NULL PRIMARY KEY,
+            record_id TEXT NOT NULL,
+            record_revision INTEGER NOT NULL
+        ) STRICT;",
+        "CREATE TABLE IF NOT EXISTS sync_settings(
+            id INTEGER NOT NULL PRIMARY KEY,
+            remote_url TEXT NOT NULL,
+            latest_revision INTEGER NOT NULL DEFAULT 0
```

Comment:
Perhaps it is better to use key/value? like settings table? Also why do we have id? Is the intention to have multiple records here?